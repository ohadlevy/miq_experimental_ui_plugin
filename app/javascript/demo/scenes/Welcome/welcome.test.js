import React from 'react';
import { configure, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Welcome from './index';

configure({ adapter: new Adapter() });

describe('Nav component', () => {
  it('renders the navigation', () => {
    const wrapper = mount(<Welcome />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
