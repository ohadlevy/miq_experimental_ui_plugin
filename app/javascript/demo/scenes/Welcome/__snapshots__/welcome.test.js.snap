// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders the navigation 1`] = `
<Component>
  <div>
    <h1>
      Experimental UI
    </h1>
    <sub>
      Thanks for testing out the new React interface!
    </sub>
    <hr />
    <p>
      Click on the kebab to the right to navigate through the available pages.
    </p>
  </div>
</Component>
`;
